diff --git a/node_modules/react-native-system-navigation-bar/android/build.gradle b/node_modules/react-native-system-navigation-bar/android/build.gradle
index 6c4b14c..b9cd5f0 100644
--- a/node_modules/react-native-system-navigation-bar/android/build.gradle
+++ b/node_modules/react-native-system-navigation-bar/android/build.gradle
@@ -1,17 +1,3 @@
-buildscript {
-    if (project == rootProject) {
-        repositories {
-            google()
-            mavenCentral()
-            jcenter()
-        }
-
-        dependencies {
-            classpath "com.android.tools.build:gradle:7.2.1"
-        }
-    }
-}
-
 apply plugin: 'com.android.library'
 
 def safeExtGet(prop, fallback) {
@@ -49,7 +35,6 @@ repositories {
     }
     google()
     mavenCentral()
-    jcenter()
 }
 
 dependencies {
